cmd/secrets-store-csi-driver/main.go

```diff
@@ -74,6 +75,16 @@ func main() {
 
 	log.SetReportCaller(*logReportCaller)
 
+	// initialize metrics exporter before creating measurements
+	// Issue: https://github.com/open-telemetry/opentelemetry-go/issues/677
+	// this has been resolved in otel release v0.5.0
+	// TODO (aramase) update to latest version of otel and deps
+	m, err := metrics.NewMetricsExporter()
```

Comment:
This limitation was identified as part of testing - https://github.com/kubernetes-sigs/secrets-store-csi-driver/pull/323. Adding this to the PR to confirm the e2e tests validate it.